my-pern-app/frontend/src/App.tsx

```diff
@@ -14,8 +16,11 @@ function App() {
 
 
   return <>
-  <div className="text-center text-xl">{greeting}</div>
-  <h1>frontend header</h1>
+  <MovieList/>
```

Comment:
No use of React Router?

This is the backend challenge, so that's OK. But for the backend challenge you can eliminate the frontend entirely and just use httpIE to test it.